src/components/ResourceList/components/FilterControl/components/DateSelector/tests/DateSelector.test.tsx

```diff
@@ -299,7 +347,9 @@ describe('<DateSelector />', () => {
       trigger(wrapper.find(TextField), 'onChange', date);
       trigger(wrapper.find(TextField), 'onBlur');
 
-      expect(onFilterValueChangeSpy).toHaveBeenCalledWith(date);
+      expect(onFilterValueChangeSpy).toHaveBeenCalledWith(
+        '2019-08-22T18:00:00.000Z',
```

Comment:
I feel that the expected values should be built using your mocked timezone offset